### PR TITLE
[DC-975] Drop Rows with 0 Source and Standard Concept_IDs from Clean

### PR DESCRIPTION
**Pull request by amcgrenera-vumc**

### CODE REVIEW

File: tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/drop_zero_concept_ids_test.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    def setUp(self):
+        self.project_id = 'test_project'
+        self.dataset_id = 'test_dataset'
+        self.sandbox_id = 'test_sandbox'
+        self.client = None
+
+        self.query_class = DropZeroConceptIDs(self.project_id, self.dataset_id,
+                                              self.sandbox_id)
+
+        self.assertEquals(self.query_class.project_id, self.project_id)
```
Comment (dcampbell-vumc) [inline tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/drop_zero_concept_ids_test.py:67]: Use `self.assertEqual()` for these as `assertEquals()` is being deprecated

File: data_steward/cdr_cleaner/cleaning_rules/drop_zero_concept_ids.py [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+{{unique_identifier}} NOT IN (
+SELECT {{unique_identifier}}
+FROM {{project}}.{{sandbox_dataset}}.{{sandbox_table}}
+)
+""")
+
+
+class DropZeroConceptIDs(BaseCleaningRule):
+    """
+    Apply value ranges to ensure that values are reasonable and to minimize the likelihood
```
[the remark this answers is not in the document]
Comment (amcgrenera-vumc) [reply data_steward/cdr_cleaner/cleaning_rules/drop_zero_concept_ids.py:121]: good catch, thank you!

File: data_steward/cdr_cleaner/cleaning_rules/drop_zero_concept_ids.py [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+        """
+        Function to run any data upload options before executing a query.
+        """
+        pass
+
+    def setup_validation(self, client):
+        """
+        Run required steps for validation setup
+        """
+        pass
```
[the remark this answers is not in the document]
Comment (amcgrenera-vumc) [reply data_steward/cdr_cleaner/cleaning_rules/drop_zero_concept_ids.py:196]: got it, thank you!

**Comment by amcgrenera-vumc** [comment]
@dcampbell-vumc thank you for your review, I've gone ahead and made adjustments, please take a look when you get a moment.

**Pull request merged**